layouts/partials/head.html

```diff
@@ -22,9 +21,7 @@
 {{ end -}}
 {{ partialCached "favicons.html" . }}
 <title>{{ if .IsHome }}{{ .Site.Title }}{{ else }}{{ with .Title }}{{ . }} | {{ end }}{{ .Site.Title }}{{ end }}</title>
-{{- template "_internal/opengraph.html" . -}}
 {{- template "_internal/schema.html" . -}}
-{{- template "_internal/twitter_cards.html" . -}}
```

Comment:
Removing these internal template calls as we are customizing them later in the same layout file.